docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,292 @@
+]
+
+2021-12-01 18:26:42,384 [ INFO] [main] [c.h.c.LifecycleService]: [172.18.0.2]:5701 [blue] [{full-version}] [172.18.0.2]:5701 is STARTED
+----
+
+== Step 2. Start the Green Enterprise Cluster
+
+Start another local single-member cluster called `green`.
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --network hazelcast-network \
+    --rm \
+    -e HZ_CLUSTERNAME=green \
+    -e HZ_LICENSEKEY=<your license key> \ <1>
+    -p 5701:5701 hazelcast/hazelcast-enterprise:{full-version}
+----
+<1> Replace the `<your license key>` placeholder with your Hazelcast Enterprise license key.
+
+See the `green` cluster is formed:
+
+[source,shell]
+----
+2021-12-01 18:28:46,299 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.3]:5701 [green] [5.0] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.3]:5701 - 72f5520c-8c27-4501-9199-a8da6b58c0b4 this
+]
+
+2021-12-01 18:28:46,321 [ INFO] [main] [c.h.c.LifecycleService]: [172.18.0.3]:5701 [green] [5.0] [172.18.0.3]:5701 is STARTED
+----
+
+Now, you have two separate Hazelcast Enterprise clusters running on your local.
+
+== Step 3. Configure the Client
+
+In this step, you'll configure a client, so that it initially connects to the `blue` cluster, and when
+the `blue` cluster is down, it automatically connects to the `green` cluster.
+
+For this, you need to create two client configurations for the same client, and pass these to a failover configuration.
+
+. Create the client configuration file for the `blue` cluster, called `client-blue.yaml` (or `client-blue.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client:
+  cluster-name: blue
+  network:
+    cluster-members:
+      - 172.18.0.2
+----
+--
+
+XML::
++
+[source,shell]
+----
+<hazelcast-client>
+    <cluster-name>blue</cluster-name>
+    <network>
+        <cluster-members>
+            <address>172.18.0.2</address>
+        </cluster-members>
+    </network>
+</hazelcast-client>
+----
+====
+. Create the client configuration for the `green` cluster, called `client-green.yaml` (or `client-green.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client:
+  cluster-name: green
+  network:
+    cluster-members:
+      - 172.18.0.3
+----
+--
+
+XML::
++
+[source,shell]
+----
+<hazelcast-client>
+    <cluster-name>green</cluster-name>
+    <network>
+        <cluster-members>
+            <address>172.18.0.3</address>
+        </cluster-members>
+    </network>
+</hazelcast-client>
+----
+====
+. Create the client failover configuration by passing the above two client configurations.
+Name of this configuration file must be `hazelcast-client-failover.yaml` (or `hazelcast-client-failover.xml`).
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+[source,shell]
+----
+hazelcast-client-failover:
+  try-count: 4 <1>
+  clients:
+    - client-blue.yaml
+    - client-green.yaml
+----
+<1> Count of the client's reconnection attempts to the cluster.
```

Comment:
```suggestion
<1> Number of times that the client will try to reconnect to each cluster before shutting down.
```

We can move this below the tabs to avoid duplication.